config/initializers/delayed_job_config.rb

```diff
@@ -10,6 +10,17 @@
 
 Delayed::Worker.delay_jobs          = !Rails.env.test?
 
+if ENV['HEROKU'].present?
+  Delayed::Worker.logger = Logger.new(STDOUT)
+else
+  Delayed::Worker.logger = Logger.new(File.join(Rails.root, 'log', 'delayed_job.log'))
+end
+
+if Delayed::Worker.delay_jobs && ($PROGRAM_NAME =~ /delayed_job/)
+  ActiveRecord::Base.logger = Delayed::Worker.logger
+  Rails.logger = Delayed::Worker.logger
```

Comment:
In AAPC this code change is working. Now are able to see all logs created in dj in  `delayed_job.log` and I tested this feature in heroku. So going to merge it.